linkerd/telemeter/usage/src/test/scala/io/buoyant/linkerd/telemeter/UsageDataTelemeterTest.scala

```diff
@@ -0,0 +1,29 @@
+package io.buoyant.linkerd.telemeter
+
+import com.twitter.finagle.util.LoadService
+import io.buoyant.config.Parser
+import io.buoyant.linkerd.{TracerConfig, TracerInitializer}
+import io.buoyant.telemetry.{TelemeterConfig, TelemeterInitializer}
+import org.scalatest.FunSuite
+
+class UsageDataTelemeterTest extends FunSuite {
```

Comment:
I think UsageDataTelemeter could use an e2e test where you spin up a Service and receive data from it.